relays/client-bridge-hub-wococo/src/runtime_wrapper.rs

```diff
@@ -16,11 +16,91 @@
 
 //! Types that are specific to the BridgeHubWococo runtime.
 
-// We reuse everything from rococo runtime wrapper
-pub type Call = relay_bridge_hub_rococo_client::runtime::Call;
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+
+pub use bp_header_chain::BridgeGrandpaCallOf;
+pub use bp_parachains::BridgeParachainCall;
+pub use bp_runtime::calls::SystemCall;
+pub use bridge_runtime_common::messages::BridgeMessagesCallOf;
+
+// TODO:check-parameter - check SignedExtension
```

Comment:
Can you, please, file an issue for this TODO and leave the reference here. Same for other similar TODOs - this way we won't forget to actually check it when the PR will be merged :)

Reply:
I guess, this TODO was about those `generate_bridge_reject_obsolete_headers_and_messages / BridgeRejectObsoleteHeadersAndMessages`, so this can be removed here

Reply:
I opened #1748 for this